database-postgresql/seed.js

```diff
@@ -0,0 +1,62 @@
+// helper functions to seed data
+const faker = require('faker');
+const request = require('request');
+const client = require('./connection');
+const fs = require('fs');
+
+// each trail id from 1 - 100 is represented in the trailPhotos database
+// not every user has a photo
+
+// enter number of mock trails and maximum number of photos per trail below
+var numSampleTrails = 100;
+var maxNumPhotosPerTrail = 5;
+
+
+// helper functions
+var getRandomIntInclusive = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+var insertionFactory = (trailId, i) => {
+  return new Promise ((resolve, reject) => {
+
+    var requestPromise = new Promise((resolve, reject) => { // async process that grabs random nature photo
+      request('https://loremflickr.com/700/520/nature', (err, res) => {
+        if(err) throw err;
```

Comment:
Same as above, where is this throwing to? 

Is it necessary to throw? Possible to just have 1 static image in a public folder act as a fallback.

If the api call fails, all that's really needed is an image, right?

Reply:
Make your service resilient. 

Errors should be developer facing as much as possible. Only expose errors to your user if there is no possible fallback.